File: src/main/java/dev/interfiber/karpet/KarpetLauncher.java
package dev.interfiber.karpet;

import dev.interfiber.karpet.server.init.ServerBootstrap;
import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;


public class KarpetLauncher {
    // Log4j
    private static final Logger LOG = LogManager.getLogger(KarpetLauncher.class);
    
    public static void main(String[] args) {
        LOG.info("Karpet version 1.0-SNAPSHOT");
        LOG.info("Starting karpet server...");
        ServerBootstrap.StartServer();
    }
}

File: src/main/kotlin/KarpetLauncher.kt
import dev.interfiber.karpet.server.init.ServerBootstrap
import mu.KotlinLogging
private val logger = KotlinLogging.logger {}

fun main() {
    logger.info("Karpet version 1.0-SNAPSHOT");
    logger.info("Starting karpet server...");
    ServerBootstrap().startServer();
}